test-utils/src/main/java/apoc/util/TestContainerUtil.java

```diff
@@ -129,24 +131,32 @@ private static Neo4jContainerExtension createNeo4jContainer(
             e.printStackTrace();
         }
 
-        for (ApocPackage apocPackage : apocPackages) {
-            if (apocPackage == ApocPackage.CORE) {
-                projectDir = coreDir;
-            } else {
-                projectDir = extendedDir;
-            }
+        boolean testDockerBundle = System.getProperty(APOC_TEST_DOCKER_BUNDLE).equals("true");
+
+        if (!testDockerBundle) {
+            for (ApocPackage apocPackage : apocPackages) {
+                if (apocPackage == ApocPackage.CORE) {
+                    projectDir = coreDir;
+                } else {
+                    projectDir = extendedDir;
+                }
+
+                executeGradleTasks(projectDir, "shadowJar");
 
-            executeGradleTasks(projectDir, "shadowJar");
+                copyFilesToPlugin(
+                        new File(projectDir, "build/libs"),
+                        new WildcardFileFilter(Arrays.asList("*-extended.jar", "*-core.jar")),
+                        pluginsFolder);
+            }
+        }
 
-            copyFilesToPlugin(
-                    new File(projectDir, "build/libs"),
-                    new WildcardFileFilter(Arrays.asList("*-extended.jar", "*-core.jar")),
-                    pluginsFolder);
+        if (testDockerBundle && apocPackages.contains(ApocPackage.EXTENDED)) {
+            throw new IllegalArgumentException("You cannot run these tests with apoc extended bundled inside"
+                    + "the docker container because only apoc core comes bundled in those");
```

Comment:
```suggestion
            throw new IllegalArgumentException("You cannot run these tests with apoc extended bundled inside "
                    + "the docker container because only apoc core comes bundled in those");
```